SCOPE.md

```diff
@@ -0,0 +1,32 @@
+# AsyncAPI Scope
+
+AsyncAPI is a specification for defining message-driven APIs. We're on a mission to standardize message-based communication and increase interoperability of the different types of systems out there.
+
+## Scope
+
+The following is a list of features the AsyncAPI specification and its official tooling should offer:
+
+- Provide enough information to connect and consume message-driven APIs. An AsyncAPI document should provide all your software needs to connect to a given API and start publishing and/or consuming messages from it.
+- Offer a framework for validating message payloads and headers.
+- Allow you to strictly define your message-driven API in a machine-readable format.
+- Provide mechanisms for mapping the information contained in an AsyncAPI document to the major messaging protocols.
+- Protocol-specific features, such as queues, exchanges, policies, partitions, etc; always via protocol mappers.
+- Provide a standard way to create specification extensions and promote their discoverability. 
+- Offer mechanisms to define message schemas in different formats. Namely: JSON Schema, OpenAPI schema (_slightly different than JSON Schema Draft 04_), Protobuf, Avro, etc.
+- Provide parsers for all the major programming languages, so it's easier for others to create tooling.
```

Comment:
Perhaps separate the definition spec from the tooling, as not to create confusion like what we saw with Swagger vs. OAS?!

Reply:
I'd like to keep it part of the same thing. The problem between Swagger and OAS is that one is part of a company and the other part of a foundation. I want to keep both together, always. Most people don't care about specs, but they do about the tooling. Ultimately is what you use. That's one of the reasons it's being hard for the OAS folks to make people say OpenAPI and not Swagger.

Reply:
Fair enough. May I suggest then to rework the scope and such and call out the spec + tooling as two separate offerings as part of the initiative? The bullets indicate this but aren't grouped, so it becomes confusing. I could see readers asking "is it a spec? Is it a solution? Is it tools?" Simply reorganizing and clarifying the intent will go far, I think.

Reply:
Agree. It should be clear.